ios/keyframes-sample-macos/swift/keyframes-sample-macos/keyframes-sample-macos-Bridging-Header.h

```diff
@@ -0,0 +1,7 @@
+//
```

Comment:
Header is wrong.
And also the iOS swift sample is not using bridging header any more. Let's follow that pattern.